pkg/rpm/unit.spec.in

```diff
@@ -1,5 +1,5 @@
 # distribution specific definitions
-%define use_systemd (0%{?rhel} && 0%{?rhel} >= 7) || (0%{?suse_version} >= 1315)
+%define use_systemd (0%{?rhel} && 0%{?rhel} >= 7) || (0%{?fedora} && 0%{?fedora} >= 18) || (0%{?suse_version} >= 1315)
```

Comment:
I believe `0%{?fedora} >= 18` is enough here. Also, I'd place this after `suse_version` check just to honor original ordering of platforms as they've been added to unit spec.